lib/board.rb

```diff
@@ -0,0 +1,59 @@
+class Board
+    attr_accessor :cells
+
+    def initialize
+        self.cells = Array.new
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.